mne/report.py

```diff
@@ -698,7 +703,13 @@ def _validate_input(self, items, captions, section, comments=None):
         if not isinstance(captions, (list, tuple)):
             captions = [captions]
         if not isinstance(comments, (list, tuple)):
-            comments = [comments]
+            if comments is None:
+                comments = [comments] * len(captions)
+            else:
+                comments = [comments]
+        elif not len(comments) == len(items):
```

Comment:
so whenever captions and comments have same length, we throw an error which says that captions must have same length? Then the user goes through another iteration and gets another error which says that comments and report items must have same length? Should we combine it into a single check to avoid this iteration?